packages/client/src/game/data/constants.ts

```diff
@@ -12,26 +12,29 @@ import { Color } from './colors';
 import { classicDesignSprite } from './spriteCoordinates';
 
 export const concreteCells: Cell[] = [
-  Cell.CONCRETE,
-  Cell.CONCRETE_TOP,
-  Cell.CONCRETE_RIGHT,
-  Cell.CONCRETE_BOTTOM,
-  Cell.CONCRETE_LEFT,
-  Cell.CONCRETE_LEFT_BOTTOM,
-  Cell.CONCRETE_RIGHT_BOTTOM,
+  //mapmanager
+  Cell.Concrete,
+  Cell.ConcreteTop,
+  Cell.ConcreteRight,
+  Cell.ConcreteBottom,
+  Cell.ConcreteLeft,
+  Cell.ConcreteBottomLeft,
+  Cell.ConcreteBottomRight,
 ];
 
 export const brickCells: Cell[] = [
-  Cell.BRICK,
-  Cell.BRICK_TOP,
-  Cell.BRICK_RIGHT,
-  Cell.BRICK_BOTTOM,
-  Cell.BRICK_LEFT,
-  Cell.BRICK_LEFT_BOTTOM,
-  Cell.BRICK_RIGHT_BOTTOM,
+  //mapmanager
+  Cell.Brick,
+  Cell.BrickTop,
+  Cell.BrickRight,
+  Cell.BrickBottom,
+  Cell.BrickLeft,
+  Cell.BrickBottomLeft,
+  Cell.BrickBottomRight,
 ];
 
 export const screenClasses = {
+  //screens
```

Comment:
Несмотря на то, что мы это обсуждали на последнем синке, такие комментарии следует расписывать поподробнее, а то суть может потом забыться, когда дело дойдёт до выполнения соответствующей задачи. Лучше ставить пометку TODO, чтобы можно было потом легко найти похожие фрагменты, а сам комментарий - поместить перед строкой `export const screenClasses`.

```suggestion
  //TODO: переместить эту переменную в папку screens
  export const screenClasses = {
```